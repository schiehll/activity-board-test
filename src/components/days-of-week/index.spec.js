import React from 'react'
import renderer from 'react-test-renderer'
import DaysOfWeek from 'components/days-of-week'

describe('DaysOfWeek', () => {
  it('should render as expected', () => {
    const tree = renderer.create(<DaysOfWeek />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
