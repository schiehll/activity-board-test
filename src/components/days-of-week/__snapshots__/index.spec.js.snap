// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DaysOfWeek should render as expected 1`] = `
<div
  className="sc-bwzfXH bwZAkW"
>
  <span>
    mon
  </span>
  <span>
    wed
  </span>
  <span>
    fri
  </span>
</div>
`;
